config/smarty.config.inc.php

```diff
@@ -111,6 +112,11 @@ function smartyShowObject($params, &$smarty)
     return Tools::p($params['var']);
 }
 
+function smartyDump($params, &$smarty)
+{
+    return Tools::dump($params['var']);
```

Comment:
`Tools::dump()` does not seem to exist. Don't you mean to use `dump()` directly?